doc/man3/ASN1_STRING_length.pod

```diff
@@ -28,11 +28,11 @@ ASN1_STRING_to_UTF8 - ASN1_STRING utility functions
 
 These functions allow an B<ASN1_STRING> structure to be manipulated.
 
-ASN1_STRING_length() returns the length of the content of I<x>.
+ASN1_STRING_length() returns the length of the content of I<x>. I<x> B<MUST NOT> be B<NULL>.
```

Comment:
Please revert the `NULL`->`B<NULL>` changes. @InfoHunter was wrong, NULL should not be put in bold.